Week3/homework/step2-3.js

```diff
@@ -31,8 +34,12 @@ function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
 
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  if (num <= 0) return result;
+
+  do {
+    result += str;
+    --num;
+  } while (num > 0);
```

Comment:
Yes, I made it intentionally because, if `number < 0`, `for loop` and the `while loop `won't be executed at all. But `do-while loop` will be executed at least once, so I need to check the value before `do-while loop`, but others have this check internally.